test/azure/pom.xml

```diff
@@ -65,6 +65,13 @@
       <artifactId>slf4j-simple</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>codegen-tests</artifactId>
+      <version>1.0.0-SNAPSHOT</version>
+      <type>test-jar</type>

```

Comment:
This lets us share RunServerListener with the other modules even though it lives under src/test/java.